packages/analytics-js/src/components/eventManager/utilities.ts

```diff
@@ -247,7 +247,9 @@ const getEnrichedEvent = (
       sessionId: state.session.sessionInfo.value.id || undefined,
       sessionStart: state.session.sessionInfo.value.sessionStart || undefined,
       consentManagement: {
-        deniedConsentIds: clone(state.consents.data.value.deniedConsentIds),
+        deniedConsents: clone(state.consents.data.value.deniedConsents),
+        // For backward compatibility with older rudder-server versions
+        deniedConsentIds: clone(state.consents.data.value.deniedConsents),
```

Comment:
why we changed that? We explicitly had the suffic "Ids" to clearly state that the values passed by the user should be the ids for the categories/puproses/whatever for their third party consent manager and not the lables/names.

This stands as a constant reminder for developers that these should be the Ids.

It is more appropriate to rename the allowedConcents to allowedConcentIds that changing deniedConsentIds

Reply:
I was unaware of the decision to put the `ids` suffix explicitly, but it felt more like it was meant for OneTrust, as it supports both names and IDs.

It might be confusing when integrating a CMP that doesn't have the concept of IDs per se. 

So, I thought, using a generic name like `consents` across the board (destinations config, UI, SDK event's context, load options, etc.) would help clear any ambiguity.

Reply:
The same issues came up on firday with ketch. I imagine all consent managers will have an Id and a human redable label. We always need the ID

Reply:
Ok, in that case, let's use the IDs suffix everywhere. I'll modify the design doc and other things a bit later.

Reply:
Done.